firebase-ai/src/main/kotlin/com/google/firebase/ai/type/LiveSession.kt

```diff
@@ -419,6 +450,9 @@ internal constructor(
             )
           }
           is LiveServerContent -> {
+            if (it.outputTranscription != null || it.inputTranscription != null) {
```

Comment:
```suggestion
            if (it.inputTranscription != null || it.outputTranscription != null) {
```

Swap them to match the expression below